tests/e2e/tracks-page.test.ts

```diff
@@ -103,10 +81,10 @@ test.describe('Tracks Page', () => {
       await searchInput.fill(searchTerm);
 
       await page.waitForTimeout(500);
```

Comment:
It is not a good practice to leave arbitrary timeouts inside of tests.
If for some reason the server responds a bit longer, the test will fail.
I would recommend just waiting for an element you want to appear (in this case it's the `noTracksMessage`)
If it takes longer to appear, you can add a custom time limit to [this specific element check](https://playwright.dev/docs/test-timeouts#specify-expect-timeout-for-a-single-assertion)